ports/opentelemetry-cpp/vcpkg.json

```
@@ -23,7 +23,10 @@
   ],
   "features": {
     "elasticsearch": {
-      "description": "Whether to include the Elasticsearch Client in the SDK"
+      "description": "Whether to include the Elasticsearch Client in the SDK",
+      "dependencies": [
+        "curl"
```

Don't block users' opt-out from default features:
```suggestion
        {
          "name": "curl",
          "default-features": false
        }
```

---

OK, pushed an update